modules/core/src/main/java/org/apache/ignite/internal/processors/cache/WalStateManager.java

```diff
@@ -304,6 +305,12 @@ public boolean prohibitWALDisabling() {
             String.format("Cache WAL mode cannot be changed within lock or transaction " +
                     "[cacheNames=%s, walEnabled=%s]", cacheNames, enabled));
 
```

Comment:
```suggestion
     // TODO: IGNITE-13976 WAL disable/enable with node restarts results in mismatching state, data loss
```

Reply:
There's a link in the text below. But I can also add this comment. What do you think?

Reply:
Ilya, from my point of view we should add such comment due to we can forget to remove warnings when the issue will be fixed.

Reply:
But as you wish. Let's proceed anyway.

Reply:
I think somebody will point out that the warning is still shown even through the issue is resolved. I will proceed.